src/main/java/uk/gov/companieshouse/api/strikeoffobjections/processor/ObjectionProcessor.java

```diff
@@ -74,7 +74,18 @@ public void process(Objection objection, String httpRequestId)
 
         // TODO update status to CHIPS_SENT
 
-        // TODO OBJ-172 send internal email
+        // send internal email
+        try {
+            CompanyProfileApi companyProfile = this.companyProfileService.getCompanyProfile(objection.getCompanyNumber(), httpRequestId);
```

Comment:
Now that you retrieve company profile here, you can remove the duplicate call lower down (line 94).